### PR TITLE
[Platform]: Add ClinVar widget to variant page

### PR DESCRIPTION
**Pull request by gjmcn**
## Description

Add ClinVar widget to variant page.

**Issue:** [#3318](https://github.com/opentargets/issues/issues/3318)
**Preview:** https://deploy-preview-447--ot-platform.netlify.app/variant/4_1804392_G_A

Note: builds on PR https://github.com/opentargets/ot-ui-apps/pull/441.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Checked on dev.

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/sections/src/variant/EVA/Body.tsx [hunk 1]
```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
...
-    {
-      id: "reviewStatus",
-      label: "Review status",
-      renderCell: ({ confidence }) => (
+    filterValue: ({ alleleOrigins }) => (alleleOrigins?.join(" ") || ""),
+  },
+  {
+    id: "reviewStatus",
+    label: "Review status",
+    renderCell: ({ confidence }) => (
```
Comment (chinmehta) [inline packages/sections/src/variant/EVA/Body.tsx:144]:
```
     if(!confidence) return naLabel;
      return (<Tooltip title={confidence}>
          <ClinvarStars num={clinvarStarMap[confidence]} />
      </Tooltip>)
```

File: packages/sections/src/variant/EVA/Body.tsx [hunk 2]
```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
...
+    filterValue: ({ clinicalSignificances }) => {
+      return clinicalSignificances?.join(" ") || "";
+    }
+  },
+  {
+    id: "allelicRequirements",
+    label: "Allele origin",
+    renderCell: ({ alleleOrigins, allelicRequirements }) => {
+      if (!alleleOrigins || alleleOrigins.length === 0) return naLabel;
+      if (allelicRequirements) {
```
Comment (chinmehta) [inline packages/sections/src/variant/EVA/Body.tsx:116]:
```
 if (!alleleOrigins || alleleOrigins.length === 0) return naLabel;
      let displayElement = {alleleOrigins.map(a => sentenceCase(a)).join("; ")}
      if (allelicRequirements) {
        displayElement = 
          <Tooltip title={} >
            {displayElement}
          </Tooltip>
      }
      return displayElement;
    },
```

File: packages/sections/src/variant/EVA/Body.tsx [hunk 3]
```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
...
+      if (!clinicalSignificances || clinicalSignificances.length === 0) {
         return naLabel;
-      },
-      filterValue: ({ clinicalSignificances }) => clinicalSignificances.join(),
+      }
+      if (clinicalSignificances.length === 1) {
+        return sentenceCase(clinicalSignificances[0]);
+      }
+      return (
+        <ul
```
Comment (chinmehta) [inline packages/sections/src/variant/EVA/Body.tsx:94]: why do we need `ul` and `li` tags here ? i can find example in preview link.

**Review by chinmehta — CHANGES_REQUESTED** [review]

File: packages/sections/src/variant/EVA/Body.tsx (hunk 1 above)
Comment (gjmcn) [reply packages/sections/src/variant/EVA/Body.tsx]: Done

File: packages/sections/src/variant/EVA/Body.tsx (hunk 2 above)
Comment (gjmcn) [reply packages/sections/src/variant/EVA/Body.tsx:115]: Done

File: packages/sections/src/variant/EVA/Body.tsx [hunk 4]
```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
...
-            </ul>
-          );
+    exportValue: ({ disease }) => disease.name,
+    filterValue: ({ disease }) => disease.name,
+  },
+  {
+    id: "studyId",
+    label: "ClinVar ID",
+    renderCell: ({ studyId }) =>
+      studyId ? (
```
[the remark this answers is not in the document]
Comment (gjmcn) [reply packages/sections/src/variant/EVA/Body.tsx]: Done

File: packages/sections/src/variant/EVA/Body.tsx [hunk 5]
```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
...
-                      {cohortPhenotypes.map(cp => (
-                        <div key={cp}>{cp}</div>
-                      ))}
-                    </Typography>
-                  </>
-                ) : (
-                  ""
-                )}
+                {
+                  diseaseFromSource &&
```
[the remark this answers is not in the document]
Comment (gjmcn) [reply packages/sections/src/variant/EVA/Body.tsx]: Simplified similar to above comments. Logic inside the tooltip is a little ugly but not sure alternatives are any cleaner?

File: packages/sections/src/variant/EVA/Body.tsx (hunk 3 above)
Comment (chinmehta) [reply packages/sections/src/variant/EVA/Body.tsx:92]: we can replace `ul` and `li` with `Box` as it will take care of showing element in next line.

Comment (gjmcn) [reply packages/sections/src/variant/EVA/Body.tsx]: Done

**Review by chinmehta — APPROVED** [review]

**Pull request merged**